.gitignore

```diff
@@ -6,3 +6,4 @@
 venv/
 .pytest_cache/
 **__pycache__/
+db.sqlite3
```

Comment:
There is an extra space before `db.sqlite3`. This might prevent the file from being ignored by Git. Consider removing the space to ensure it is correctly ignored.